index.go

```diff
@@ -0,0 +1,32 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package main
+
+import (
+	"encoding/json"
+	"net/http"
+	"time"
+)
+
+type indexData struct {
+	ServiceName string `json:"service.name"`
+	Version     string `json:"version"`
+}
+
+func indexHandler(cacheTime time.Duration) (func(w http.ResponseWriter, r *http.Request), error) {
+	data := indexData{
+		ServiceName: "package-registry",
+		Version:     "0.4.0",
```

Comment:
Nit: Could have this as a constant in the main.go file?

Reply:
Fixed